include/bluetooth/mesh/light_ctrl_cli.rst

```diff
@@ -0,0 +1,29 @@
+.. _bt_mesh_light_ctrl_cli_readme:
+
+Light Lightness Control Client
+##############################
+
+The Light Lightness Control (LC) Client configures and interacts with the Light LC Server.
+
+Contrary to the Server model, the Client only creates a single model instance in the mesh composition data.
```

Comment:
Why is this? Is it because of its size? I'm asking, because the first part puts an emphasis on "contrary to the server model", so the question is why and what the difference is.

Reply:
This comes from the spec. I can align it with the template.